test/test_optim.py

```diff
@@ -1786,6 +1786,49 @@ def test_fused_optimizer_raises(self):
             with self.assertRaisesRegex(RuntimeError, "`fused` does not support `differentiable`"):
                 optimizer_ctor([torch.empty((), device="cuda")], differentiable=True, fused=True)
 
+    def test_lars(self):
+        # ASK: What's the reason behind two identical calls? (See SGD tests)
```

Comment:
sorry, which two are you referring to here? The optim tests are certainly up for a refactor, so there might just be actual redundancies we can get rid of

Reply:
I think the comment is no longer valid.